### DISCUSSION TITLE
Feature Request: Publishing via MQTT the IEEE address or some other unique ID/number

### DISCUSSION THREAD

**Discussion by TheNutHouse84** [Koenkk/zigbee2mqtt#16544] [General]
I log various sensor readings to a SQL database. It would be helpful if Z2M would publish the IEEE address or some other unique id number for each paired device. 

For example: Hubitat creates an incrementing integer number each time a device is paired that is unique to that device (if it is removed then re-paired it gets a new number). That number is then published in the log output. 

Optionally the ability to for an additional entry like the device Friendly Name. Perhaps a "Friendly Number" entry that can be edited.

**Comment by sjorge** [comment]
You can subscribe to the bridge information, then just map the friendly_name to the ieee? 

You can also set `include_device_information` https://www.zigbee2mqtt.io/guide/configuration/mqtt.html#server-connection and it will include that info on each published message too.

**Reply by TheNutHouse84** [reply]
Thank you.
